plugins/adapters/modbus/srcCxx/LibModbusClient.cxx

```diff
@@ -46,6 +46,8 @@ void LibModbusClient::connect(const std::string& ip, unsigned int port)
                 + "> " + modbus_strerror(errno));
         throw std::runtime_error(error);
     }
+    std::string ip_address_ = ip;
+    uint16_t port_number_ = port;
```

Comment:
Aren't you redefining these variables? Did you mean

```cpp
ip_address_ = ip;
port_number_ = port;
```

Reply:
nice catch! Although those variables are not used for any functional feature, they are not correctly assigned.